dbms/src/AggregateFunctions/AggregateFunctionMinMaxWindow.h

```diff
@@ -0,0 +1,231 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <AggregateFunctions/AggregateFunctionMinMaxAny.h>
+#include <AggregateFunctions/IAggregateFunction.h>
+#include <Columns/ColumnDecimal.h>
+#include <Columns/ColumnString.h>
+#include <Columns/ColumnVector.h>
+#include <Common/typeid_cast.h>
+#include <DataTypes/IDataType.h>
+#include <IO/ReadHelpers.h>
+#include <IO/WriteHelpers.h>
+#include <common/StringRef.h>
+
+#include <cassert>
+#include <set>
+
+namespace DB
+{
+template <typename T>
+struct SingleValueDataFixedForWindow : public SingleValueDataFixed<T>
```

Comment:
why this struct need inherit `SingleValueDataFixed`

Reply:
I think `SingleValueDataFixedForWindow` can not just inherit the ser/deser functions from `SingleValueDataFixed`

Reply:
> why this struct need inherit `SingleValueDataFixed`

we now delete the inheritence.